array-manipulation-2/src/drop.js

```diff
@@ -0,0 +1,12 @@
+/* exported drop */
+function drop(array, count) {
+  var newArray = [];
+  for (var i = count; i < array.length; i++) {
+    if (array[i] === newArray[i]) {
+      return newArray;
```

Comment:
You should be able to delete these lines, because the main condition is `i < array.length`.